bundles/org.openhab.binding.growatt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -589,20 +591,29 @@
 		<item-type>Number:Dimensionless</item-type>
 		<label>Status Code</label>
 		<category>Status</category>
+		<tags>
+			<tag>Status</tag>
+		</tags>
```

Comment:
Things are moving fast, so i might have missed this rule change, but AFAICT atleast two tags are needed. Property + Point.

Reply:
I believe we said that it MUST have a Point and it SHOULD have a Property.

https://next.openhab.org/docs/developer/bindings/semantic-tags.html#d-rules-for-extending-point-tags

The reason for not having a Property is that sometimes it is simply not possible to define one e.g. a status information code or text which could have arbitrary meaning.

Reply:
Yep that changed. Not sure if i support that rule, but as it seems merged and agreed on, let's continue.